### PR TITLE
chore: add url to documentFetch span

### PR DESCRIPTION
**Pull request by tchoi1131**
## Which problem is this PR solving?
Currently, the documentFetch span doesn't have the HTTP_URL, which exist in the documentLoad span. The purpose of this PR is to add the HTTP_URL to the documentFetch span.

## Short description of the changes
Use fetchSpan.setAttribute to set the HTTP_URL.

## Checklist
- [x] A manual test to test the new attributes are added to the documentFetch span
image

### CODE REVIEW

File: plugins/web/opentelemetry-instrumentation-document-load/src/instrumentation.ts
Comment (blumamir) [inline plugins/web/opentelemetry-instrumentation-document-load/src/instrumentation.ts:117]: Is it possible to add a unit-test for that?

Comment (tchoi1131) [reply plugins/web/opentelemetry-instrumentation-document-load/src/instrumentation.ts:117]: Thanks for the comment. I tried to add a unit test. But, looks like the unit test in the main branch is broken if run locally before my change. I tried to run "npm run test:browser" and "npm run tdd". Both are failed with the following message. Is it because of my OS or some other settings? Do you have the idea of what went wrong?
image

I just added the unit test. But, I couldn't run it locally to verify it.

Comment (blumamir) [reply plugins/web/opentelemetry-instrumentation-document-load/src/instrumentation.ts:117]: @MSNev maybe you know how to make it work?

@tchoi1131  Thanks for your efforts.
Sorry but I have very little experiance with browser instrumentations and tooling :/

Comment (MSNev) [reply plugins/web/opentelemetry-instrumentation-document-load/src/instrumentation.ts:117]: I haven't sync'd the contrib repo for a while so I have a look later this week.

On the unhandled rejection this can happen when promises (async tests) are used and it throws, so I think the main issue is the first "data" error.

Comment (MSNev) [reply plugins/web/opentelemetry-instrumentation-document-load/src/instrumentation.ts:117]: It appears that all of the web tests are broken 👎 I've tried several work-arounds as on my machine the "temp" path (which is on a different drive (yes window)) is being included and it fails to compile.

Attempting to roll-back the webpack update from April to see it that "resolves"

**Comment by MSNev** [comment]
Test fixes
Bug #1135 
PR #1137

File: plugins/web/opentelemetry-instrumentation-document-load/src/instrumentation.ts
Comment (tchoi1131) [reply plugins/web/opentelemetry-instrumentation-document-load/src/instrumentation.ts:117]: @MSNev  Thank you so much for following up on this!

**Comment by MSNev** [comment]
@tchoi1131 It's just waiting for more approvals

**Comment by tchoi1131** [comment]
> @tchoi1131 It's just waiting for more approvals

OK. thanks for the update

**Comment by tchoi1131** [comment]
@obecny can you help review this PR? Thanks! The unit test is fixed. all tests passed.

**Review by MSNev — APPROVED** [review]

**Comment by tchoi1131** [comment]
I don't have write access. can someone help merge this to main branch? Thanks

**Review by vmarchaud — APPROVED** [review]

**Review by obecny — APPROVED** [review]
lgtm

**Comment by scheler** [comment]
@dyladan  @legendecas does this need additional approvals or otherwise can one of you please merge this PR?

**Review by legendecas — APPROVED** [review]

**Comment by legendecas** [comment]
Thank you for your contribution!

**Pull request merged**
